src/components/simple-player/simple-player.tsx

```diff
@@ -21,22 +19,32 @@ const BUFFER_DURATION = 8000;
 export const SimplePlayer: FC<SimpleVideoProps> = ({
   uri,
   posterUri,
-  paused = true,
   size,
   style,
   onError = emptyFn,
   onLoad = emptyFn
 }) => {
-  const atBootsplash = useAtBootsplash();
-  const { isLocked } = useAppLock();
+  const [paused, setPaused] = useState(false);
+
+  useEffect(() => {
+    const subscription = AppState.addEventListener('change', nextAppState => {
+      if (nextAppState !== 'active') {
+        setPaused(true);
```

Comment:
dont you need to set it back to false, when state is 'active'?

Reply:
we will play audio after unlock manually